src/pages/innovation-day/index.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: 'wasmCloud Innovation Day 2024'
+description: 'Register for wasmCloud Innovation Day 2024'
+---
+
+![wasmCloud Innovation Day](../../../static/pages/home/innovation-day/innovation-day.png)
+
+
```

Comment:
Consider removing this space..I know why it's there and you can choose to ignore.